app/dts/behaviors/key_press.dtsi

```diff
@@ -0,0 +1,19 @@
+#include <dt-bindings/zmk/keys.h>
+
+/ {
+	behaviors {
+		kp: behavior_key_press {
+			compatible = "zmk,behavior-key-press";
+			label = "KEY_PRESS";
+			usage_page = <USAGE_KEYPAD>;
+			#binding-cells = <1>;
+		};
+
+		cp: behavior_consumer_press {
```

Comment:
I tweaked this a bit to be more generic. This is *reusing* the same "key press behavior" but setting a different usage page in the instance of the driver. This reduces duplicate code, leaves us a consistent internal API we can leverage for the eventual "key position usage page" for the split stuff.